Fix the failing auto-creation of default languages in translation field setup (#22409)
directus/directus: .changeset/curvy-houses-grin.md

---
'@directus/app': patch
---

> Fixed the failing auto-creating of default languages for custom languages collections in the translation field setup

Reviewer: ```suggestion
Fixed an issue where the creation of translation fields would fail when using an existing, custom language collection
```

Author: Funnily enough, at least in my tests, it didn't fail. But yeah, maybe something along those lines... 🤔

> Fixed the failing auto-creating of default languages for custom languages collections in the translation field setup

Author: ```suggestion
Fixed the failing auto-creating of default languages for translation fields when using an existing, custom language collection.
```
